src/styles/main.scss

```diff
@@ -1,3 +1,9 @@
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
That's not reset styles, you have to write this in `normalize.scss` file

Reply:
I got only a few rows of code for normalization, so I decided to put it in main.scss, not to create a normalize file. As I know, it's a widespread practice. Would you insist on creating normalize file?

Reply:
No problem, mentors will judge us